test/runtests.jl

```diff
@@ -3,40 +3,43 @@ using SafeTestsets
 
 const GROUP = get(ENV, "GROUP", "All")
 
-const is_APPVEYOR = Sys.iswindows() && haskey(ENV,"APPVEYOR")
+const is_APPVEYOR = Sys.iswindows() && haskey(ENV, "APPVEYOR")
 
-const is_TRAVIS = haskey(ENV,"TRAVIS")
+const is_TRAVIS = haskey(ENV, "TRAVIS")
 
-const is_CI = haskey(ENV,"CI")
+const is_CI = haskey(ENV, "CI")
 
 @time begin
-  if GROUP == "All" || GROUP == "NNODE"
-      @time @safetestset "NNODE" begin include("NNODE_tests.jl") end
-  end
-  if !is_APPVEYOR && (GROUP == "All" || GROUP == "NNPDEHan")
-      @time @safetestset "NNPDEHan" begin include("NNPDEHan_tests.jl") end
-  end
-  if GROUP == "All" || GROUP == "NNPDENS"
-      @time @safetestset "NNPDENS" begin include("NNPDENS_tests.jl") end
-  end
-  if GROUP == "All" || GROUP == "NNPDE"
-      @time @safetestset "NNPDE" begin include("NNPDE_tests.jl") end
-  end
-  if GROUP == "All" || GROUP == "NeuralAdapter"
-      @time @safetestset "NeuralAdapter" begin include("neural_adapter_tests.jl") end
-  end
-  if GROUP == "All" || GROUP == "NNKOLMOGOROV"
-      @time @safetestset "NNKolmogorov" begin include("NNKolmogorov_tests.jl") end
-  end
-  if GROUP == "All" || GROUP == "NNSTOPPINGTIME"
-      @time @safetestset "NNStopping" begin include("Stopping_tests.jl") end
-  end
-  if GROUP == "All" || GROUP == "NNRODE"
+    if GROUP == "All" || GROUP == "NNHET"
+        @time @safetestset "NNHET" begin include("NNPDE_tests_het.jl") end
```

Comment:
old format dep_vars = [Q, P, Z] it should be deprecated. 
there will supported only the new format dep_vars = [Q (t), P (t, x), Z (t, y)]. 
don't need NNHET,  add these tests in  NNPDE_tests and update all test in NNPDE_tests,  [u] -> [u(x,y)]

Reply:
Are you saying I should change the tests (and probably the docs) so that, for example, all PDE Systems call the dependent variables with their indvars? PDESystem(eq,bc,domain,[x,y],[**u(x, y)**])

Reply:
yes, and  `PDESystem(eq,bc,domain,[x,y],[u])` will be deprecated

Reply:
mentioned @ChrisRackauckas